AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -520,6 +533,13 @@ - (void)selectItemAtIndexPath:(NSIndexPath *)indexPath animated:(BOOL)animated s
   [super selectItemAtIndexPath:indexPath animated:animated scrollPosition:scrollPosition];
 }
 
+- (void)waitForUpdatesDuringNextLayoutPass
+{
+  ASDisplayNodeAssertMainThread();
+  [self setNeedsLayout];
```

Comment:
@Adlai-Holler should this actually set a new layout pass as being needed?  The name doesn't currently imply that.